pytest_bdd/utils.py

```diff
@@ -110,3 +110,23 @@ def get_markers_args_using_iter_markers(node, mark_name):
 def get_markers_args_using_get_marker(node, mark_name):
     """Deprecated on pytest>=3.6"""
     return getattr(node.get_marker(mark_name), 'args', ())
+
+
+class ConfigStack(object):
```

Comment:
Do we really need to define an object to handle a stack? I'd rather use just a list and name it `config_stack`

Reply:
Changed to list